POELiveSplitComponent/Component/Timer/LoadRemoverSplitter.cs

```diff
@@ -20,6 +21,16 @@ public class LoadRemoverSplitter : IClientEventHandler
         private IZone previousZone;
         private bool labStarted = false;
 
+        readonly List<string> IzaroDeathLines = new List<string>()
```

Comment:
I'm worried about more lines being added in the future and having the feature work inconsistently. I looked at https://youtu.be/RVmRs7DWedg?t=80 and looking at your list, either that video was incomplete or they added more lines since 2017.

I'd be happy to accept this change if you could point me to an easy method of getting the exhaustive list (as I'm not a lab runner myself) or if you pinky swear to create a new PR any time you discover new lines :D.

Reply:
I got the list from running lab and noting those lines so I could stop my timer via my separate program. As far as I know those lines have existed since 2017. I've been running lab since then and I don't recall ever having noticed new lines for completing lab. If there's a list somewhere, I don't know where it is.

As far as updating it, I would certainly update it myself if those lines ever changed, and update my fork. At which point updating them here would only be a couple of clicks away.

But I'll also add, I don't mind this MR not being merged. If you're not comfortable with it, I'm perfect happy to keep these changes on my own fork. I just know a few lab runners were interested in the timer stopping when Izaro dies (as that's when the official timer stops) so I ported my Izaro death code into your plugin. And I added the reset code as that's what I was used to, having it reset when I entered plaza right before the next run (albeit with a pop up asking if I wanted to save/discard the times). ffod, who wins the standard labs every day tested my fork out and loved the QOL of the plaza reset. I imagine most lab runners would.